client/components/modals/ReactionList/Reactions.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+
+import { IUser } from '../../../../definition/IUser';
+import { useSetting } from '../../../contexts/SettingsContext';
+import Emoji from '../../Emoji';
+import ScrollableContentWrapper from '../../ScrollableContentWrapper';
+import ReactionUserTag from './ReactionUserTag';
+
+type ReactionsProps = {
+	reactions: ReactElement;
+	onClick: (e: React.MouseEvent<HTMLElement>) => void;
+};
+
+const Reactions = ({ reactions, onClick }: ReactionsProps): ReactElement => {
+	const useRealName = useSetting('UI_Use_Real_Name');
+	return (
+		<Box display='flex' flexDirection='column' minHeight='25vh'>
```

Comment:
I don't like this minHeight that much. If there are only a few reactions, it looks weird:
![image](https://user-images.githubusercontent.com/40830821/127916158-a6edc80b-2a19-4916-a9f8-227bfa5f09ad.png)

Reply:
changed to 15vh